WcaOnRails/spec/factories/inbox_persons.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :inbox_person do
+    id { (InboxPerson.maximum(:id) || 0) + 1 }
```

Comment:
A quick comment about why this is necessary might be nice. AFAIK, we haven't had to do this with our other factories.